cypress/support/pages/rooms/pageRooms.js

```diff
@@ -655,5 +646,15 @@ class Rooms {
 			.parent()
 			.should("contain", "Lesen");
 	}
+
+	seeSpeedDialOptions(...speedDialButtonOptions) {
+		speedDialButtonOptions.forEach((buttonName) => {
+			cy.get(`[data-testid="fab-${buttonName}"]`).should("be.visible");
+		});
+	}
```

Comment:
This change is needed to work with the next one together:

```suggestion
	seeSpeedDialOptions(speedDialButtonOptions) {
		speedDialButtonOptions.forEach((buttonName) => {
			cy.get(`[data-testid="fab-${buttonName}"]`).should("be.visible");
		});
	}
```